app/jobs/searchgov_urls_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class SearchgovUrlsJob < ApplicationJob
+  queue_as :searchgov
+
+  def perform(job_id, urls)
+    @time_started = Time.zone.now
+    @total_count = urls.count
+    @uploader = BulkUrlUploader.new(job_id, urls)
```

Comment:
`BulkUrlUploader` first parameter is `name`, `job_id` is confusing. Can you rename it?